docs/samples/README.md

```diff
@@ -1,61 +1,99 @@
+scale differently from the predictor if your transformer is CPU bound while predictor requires running on GPU. 
+
+| Features  | Examples |
+| ------------- | ------------- |
+| Deploy Transformer with KFServer | [Image Transformer with PyTorch KFServer](./transformer/image_transformer)  |
+| Deploy Transformer with Triton Server | [BERT Model with tokenizer](./triton/bert)  |
+
+### Deploy InferenceService with Explainer and Outlier/Drift Detector
+Model explainability answers the question: "Why did my model make this prediction" for a given instance. KFServing 
+integrates with [Alibi Explainer](https://github.com/SeldonIO/alibi) which implements a black-box algorithm by generating a lot of similar looking intances 
+for a given instance and send out to the model server to produce an explanation.
+Also in order to trust and reliably act on model predictions, it is crucial to monitor the distribution of the incoming
+requests via various different type of detectors. [Alibi Detect](https://github.com/SeldonIO/alibi-detect) checks when the distribution of incoming requests 
+is diverging from a reference distribution such as that of the training data.
+
+| Features  | Examples |
+| ------------- | ------------- |
+| Deploy Alibi Image Explainer| [Imagenet Explainer](./explanation/alibi/imagenet)  |
+| Deploy Alibi Income Explainer| [Income Explainer](./explanation/alibi/income)  |
+| Deploy Alibi Text Explainer| [Alibi Text Explainer](./explanation/alibi/moviesentiment) |
+| Deploy Alibi Outlier Detection| [Cifar outlier detector](./outlier-detection/alibi-detect/cifar10) |
+| Deploy Alibi Drift Detection| [Cifar drift detector](./drift-detection/alibi-detect/cifar10) |
+
+### Deploy InferenceService with Cloud/PVC storage
+| Feature  | Examples |
+| ------------- | ------------- |
+| Deploy Model on S3| [Mnist model on S3](./s3) |
+| Deploy Model on PVC| [Models on PVC](./pvc)  |
+| Deploy Model on Azure| [Models on Azure](./azure) |
+
+### Autoscaling
+KFServing's main serverless capability is to allow you to run inference workload without worrying about scaling your service manually once it is deployed. KFServing leverages Knative's [autoscaler](https://knative.dev/docs/serving/configuring-autoscaling/),
+the autoscaler works on GPU as well since the Autoscaler is based on request volume instead of GPU/CPU metrics which can be hard
+ to reason about. 
+ 
 [Autoscale inference workload on CPU/GPU](./autoscaling)
 
 [InferenceService on GPU nodes](./accelerators)
 
+### Canary Rollout
+Canary deployment enables rollout releases by splitting traffic between different versions to ensure safe rollout.
+
 [Canary Rollout](./rollouts)
 
+### Kubeflow Pipeline Integration
 [InferenceService with Kubeflow Pipeline](./pipelines)
 
-[InferenceService with Request/Response Logger](./logger/basic)
+### Request Batching and Request/Response Logger
```

Comment:
Are these related in any way? Logger should go much earlier in the list as a supported feature